frontend/package-lock.json

```diff
@@ -3312,6 +3313,14 @@
         "react": "^18.3.1"
       }
     },
+    "node_modules/react-icons": {
+      "version": "5.3.0",
+      "resolved": "https://registry.npmjs.org/react-icons/-/react-icons-5.3.0.tgz",
+      "integrity": "sha512-DnUk8aFbTyQPSkCfF8dbX6kQjXA9DktMeJqfjrg6cK9vwQVMxmcA3BfP4QoiztVmEHtwlTgLFsPuH2NskKT6eg==",
```

Comment:
Not sure how react-icons work, but do you think we should have this `sha` string pushed to git in plain text?

Reply:
> Not sure how react-icons work, but do you think we should have this `sha` string pushed to git in plain text?

Yes, we do not change package-lock.json. So it will be pushed as it is.